trending_deploy/deploy.py

```diff
@@ -5,7 +5,8 @@
     create_inference_endpoint,
     get_inference_endpoint,
     list_inference_endpoints,
-    model_info as get_model_info
+    model_info as get_model_info,
+    add_collection_item
```

Comment:
```suggestion
    add_collection_item,
    delete_collection_item,
```

Reply:
On `undeploy_model`: https://huggingface.co/docs/huggingface_hub/v0.30.2/en/package_reference/hf_api#huggingface_hub.HfApi.delete_collection_item, should be super simple